databricks/koalas/indexes.py

```diff
@@ -273,6 +273,10 @@ def _with_new_scol(self, scol: spark.Column) -> 'MultiIndex':
         """
         return MultiIndex(self._kdf, scol)
 
+    # TODO: This is a temporary measure to avoid confusion, so the logic should be changed later.
```

Comment:
If this is a temp fix, why do we need it in this PR?

Reply:
@HyukjinKwon Thanks for your feedback! :) As you said, I'll create each PR separately. Thank you again for your advice!

Reply:
@HyukjinKwon Thanks for your feedback! :) As you said, I'll create each PR separately. Thank you again for your advice!